Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -5996,10 +5996,12 @@ using UIClientCallback = Function<void(Ref<API::NavigationAction>&&, NewPageCall
 static void trySOAuthorization(Ref<API::NavigationAction>&& navigationAction, WebPageProxy& page, NewPageCallback&& newPageCallback, UIClientCallback&& uiClientCallback)
 {
 #if HAVE(APP_SSO)
-    page.websiteDataStore().soAuthorizationCoordinator(page).tryAuthorize(WTFMove(navigationAction), page, WTFMove(newPageCallback), WTFMove(uiClientCallback));
-#else
-    uiClientCallback(WTFMove(navigationAction), WTFMove(newPageCallback));
+    if (!page.preferences().isExtensibleSSOEnabled()) {
```

Comment:
Isn't this check backwards?

Reply:
Oh, gosh -- yes. I was experimenting with early exit if it was off, and left this in. Let me correct (thank you for catching!).